Add README for bookmarks and page-redder samples (#1101)
GoogleChrome/chrome-extensions-samples: functional-samples/sample.bookmarks/README.md

# Bookmarks Manager

> This sample demonstrates how to use [bookmarks API](https://developer.chrome.com/docs/extensions/reference/api/bookmarks) to search, add, edit and delete bookmarks.

Reviewer: ```suggestion
This sample demonstrates how to use the [Bookmarks API](https://developer.chrome.com/docs/extensions/reference/api/bookmarks) to search, add, edit, and delete bookmarks.
```

> This sample demonstrates how to use the [Bookmarks API](https://developer.chrome.com/docs/extensions/reference/api/bookmarks) to search, add, edit, and delete bookmarks.

Reviewer 2: We aren't totally consistent here but I try to avoid coming up with names for things and prefer to just use the name of the API :)

```suggestion
This sample demonstrates how to use the [chrome.bookmarks](https://developer.chrome.com/docs/extensions/reference/api/bookmarks) API to search, add, edit, and delete bookmarks.
```

Author: Yes, I agree

## Overview

> The extension provides a bookmarks manager that is accessible by clicking on the icon extension.

Reviewer: ```suggestion
The extension provides a bookmarks manager that is accessible by clicking the extension icon.
```

> The manager consists in a popup where you can manage your bookmarks.

Reviewer: ```suggestion
The manager provides a popup where you can manage your bookmarks.
```

## Running this extension

1. Clone this repository.
2. Load this directory in Chrome as an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).

> 3. Click on the icon extension.

Reviewer 2: ```suggestion
3. Click on the extension icon.
```
